src/extension/background.js

```diff
@@ -296,10 +297,9 @@ function toggle(id) {
  * @return {string} The language
  */
 function getLanguage() {
-  const preferred = navigator.languages
+  return navigator.languages
     .map((prefLang) => LANGS.find((lang) => prefLang.startsWith(lang)))
-    .filter((lang) => !!lang);
-  return preferred.length && preferred[0] !== LANGS[0] ? `/${preferred[0]}` : '';
+    .filter((lang) => !!lang)[0] || LANGS[0];
```

Comment:
what about adding a language option to the sidekick? I always hate when apps infer any language preference based on the environment.

Reply:
I think this should be a separate enhancement request. Usually people have their browser in their preferred language, and now they can even an ordered list of language preferences (i.e. Hunganrian as primary language, but english as fallback).